test/Libraries/Microsoft.Extensions.DataIngestion.Tests/Readers/MarkItDownReaderTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.TestUtilities;
+using Xunit;
+
+namespace Microsoft.Extensions.DataIngestion.Readers.Tests;
+
+[MarkItDownCondition]
+public class MarkItDownReaderTests : DocumentReaderConformanceTests
+{
+    protected override IngestionDocumentReader CreateDocumentReader(bool extractImages = false)
+        => MarkItDownConditionAttribute.IsInstalled.Value
+        ? new MarkItDownReader(extractImages: extractImages)
+        : throw new SkipTestException("MarkItDown is not installed");
+
+    protected override void SimpleAsserts(IngestionDocument document, string source, string expectedId)
+    {
+        Assert.NotNull(document);
+        Assert.Equal(expectedId, document.Identifier);
+        Assert.NotEmpty(document.Sections);
+
+        var elements = document.EnumerateContent().ToArray();
+
+        bool isPdf = source.EndsWith(".pdf", StringComparison.OrdinalIgnoreCase);
+        if (!isPdf)
+        {
+            // MarkItDown does a bad job of recognizing Headers and Tables even for simple PDF files.
+            Assert.Contains(elements, element => element is IngestionDocumentHeader);
+            Assert.Contains(elements, element => element is IngestionDocumentTable);
+        }
+
+        Assert.Contains(elements, element => element is IngestionDocumentParagraph);
+        Assert.All(elements, element => Assert.NotEmpty(element.GetMarkdown()));
+    }
+
+    // The original purpose of the MarkItDown library was to support text-only LLMs.
+    // Source: https://github.com/microsoft/markitdown/issues/56#issuecomment-2546357264
+    // It can extract images, but the support is limited to some formats like docx.
+    [ConditionalFact]
+    public override Task SupportsImages()
+        => SupportsImagesCore(Path.Combine("TestFiles", "SampleWithImage.docx"));
```

Comment:
These are pretty large binary files.  Do they belong in the repo, or should we put them in tests assets packages?

Reply:
I'm not sure if we'll run into it in the future again, but we have the *-assets repos for things like this.  They are repos specifically for static test content that might be binary.  Depending on the size of the files we might make them azdo only (ML.NET does this, because it has files larger than git permits without adding billing for Git LFS).  If you forsee us needing more data like this in the future we could set something up for extensions.